posthog/migrations/0671_survey_schedule.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.18 on 2025-02-19 01:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0670_querytabstate_querytabstate_unique_team_created_by"),
```

Comment:
the fule us 671 at least

Reply:
sorry what? 😆 

![image](https://github.com/user-attachments/assets/a3a05012-5b6c-49cc-b1ee-ef4cea9c8b6a)

Reply:
your file is called 0671_survey_schedule.py
but the dep is. 0670_querytabstate_querytabstate_unique_team_created_by
so I guess its correct and I just assumed you should repeat the name and number here because I missed the "dependencies" name, so ignore me sorry